ena_upload/ena_upload.py

```diff
@@ -654,7 +659,27 @@ def collect_tables(args):
 
     return schema_tables
 
+def get_data_file_path(data):
+    '''Return a list of data files provided as command line argument.
```

Comment:
Can we make this explicit with a new parameter? My hope is that it's easier to document and has less magic going on.
What do you think?

Reply:
My thought was that it is easier on the user-side this way, also as I see it unlikely that a single file would be submitted. But I also see your point. Having a new parameter would also require an additional check that either `--data` or `--data-list` is provided. (Is argparse allowing for that out of the box?)

Reply:
I also like more the idea of bgruening, especially to not exclude the possibility to submit only one single read file. Note that the `--data` parameter is not mandatory and people can just submit samples or other types with our tool without  data.

Reply:
OK, agreed. In fact, I overlooked the possibility that `--data` is optional if you don't submit a RUN table. So the proposed `--no-upload`  would allow to provide the full set of metadata incl. RUN but without running the upload.

Reply:
Function was removed.